presto-docs/src/main/sphinx/release/release-0.109.rst

```diff
@@ -0,0 +1,22 @@
+=============
+Release 0.109
+=============
+
+General Changes
+---------------
+
+* Add :func:`array_min` and :func:`array_max` functions.
```

Comment:
Should we put all the functions together? That's what we've done for previous releases.